test/jdk/java/awt/color/ICC_Profile/BuiltInProfileCheck.java

```diff
@@ -48,32 +48,32 @@ public static void main(String[] args) {
 
     private static void updateProfile(boolean isBuiltIn) {
         ICC_Profile builtInProfile = ICC_Profile.getInstance(ColorSpace.CS_sRGB);
-        //create a copy of the BuiltIn profile
+        // Create a copy of the built-in profile
         ICC_Profile customProfile = ICC_Profile.getInstance(builtInProfile.getData());
 
         ICC_Profile iccProfile = isBuiltIn ? builtInProfile : customProfile;
 
         byte[] headerData = iccProfile.getData(HEADER_TAG);
         int index = PROFILE_CLASS_START_INDEX;
-        //set profile class to valid icSigInputClass = 0x73636E72
+        // Set profile class to valid icSigInputClass = 0x73636E72
         headerData[index] = 0x73;
         headerData[index + 1] = 0x63;
         headerData[index + 2] = 0x6E;
         headerData[index + 3] = 0x72;
 
         if (isBuiltIn) {
             try {
-                //try updating a BuiltIn Profile, the following stmt should throw IAE
+                // Try updating a built-in profile, IAE is expected
                 iccProfile.setData(HEADER_TAG, headerData);
-                throw new RuntimeException("Test Failed ! IAE NOT thrown.");
+                throw new RuntimeException("Test Failed! IAE NOT thrown.");
             } catch (IllegalArgumentException iae) {
-                if (!iae.getMessage().equals(EXCEPTION_MSG)) {
-                    throw new RuntimeException("Test Failed ! IAE with exception msg \""
+                if (!iae.getMessage().equalsIgnoreCase(EXCEPTION_MSG)) {
```

Comment:
I think we can compare with regular `equals`. It is unlikely the message will change by letter case only. The test uses exactly the same message as the one that's thrown in `ICC_Profile.setData`, therefore the messages should match.

If the message is changed in the code for whatever reason, the message in the test will need updating, which is expected, isn't it?